### ISSUE TITLE
Migration guide from 2.3.0 to 2.4.0

### ISSUE THREAD

**Issue by michaburger** [mapbox/mapbox-maps-flutter#775]
Hi there, what do I have to do in order to migrate from 2.3.0 to 2.4.0? My app is working as of 2.3.0 and I am defining the access tokens in the gradle.properties file as well as with `MapboxOptions.setAccessToken()`. What has changed in 2.4.0 regarding the tokens? Now I get `> Could not resolve com.mapbox.maps:android:11.8.0.
     Required by:
         project :app
      > Could not resolve com.mapbox.maps:android:11.8.0.
         > Could not get resource 'https://api.mapbox.com/downloads/v2/releases/maven/com/mapbox/maps/android/11.8.0/android-11.8.0.pom'.
            > Could not GET 'https://api.mapbox.com/downloads/v2/releases/maven/com/mapbox/maps/android/11.8.0/android-11.8.0.pom'. Received status code 401 from server: Unauthorized
      > Could not resolve com.mapbox.maps:android:11.8.0.`

**Comment by evil159** [comment]
Hi @michaburger, there shouldn't be any specific steps to migrate, could you check if you have authentication setup for Mapbox Maven repository in your grade files?

**Comment by michaburger** [comment]
Yes I do. Fact is, everything works perfectly with 2.3.0. The error only happens after upgrading to 2.4.0. Something must have changed regarding the tokens, what did you change exactly and what do I have to do?

**Comment by evil159** [comment]
@michaburger We have lifted the authentication requirement for obtaining Maps SDK binaries, could you remove authentication from your maven setup and try again/

**Comment by favazHF** [comment]
I got error when trying to compile in Android with 2.4.1 version :(

**Comment by BishopSam** [comment]
@michaburger where you able to resolve it?

**Comment by Wackymax** [comment]
It looks like the build file for this library still requires the authentication. The build.gradle under Android needs to be updated.

**Comment by michaburger** [comment]
Update: I am now able to run my code with 2.5.1 too, however I still need to set SDK_REGISTRY_TOKEN with a key in my android gradle.properties file, other wise I will get error 401 and am not able to download map tiles. This is not described in the user guide as of 2.5.1 and I'm wondering it it's the idea to still specify SDK_REGISTRY_TOKEN? What kind of token should I put there, a different one than for accessing the map?
